docs/3rd-party-tools/sizechecker.md

```diff
@@ -6,7 +6,7 @@ description: Simple tool to stop downloads if disk is full with notification sup
 keywords: [disk-monitoring,space,free-space,free,disk]
 ---
 
-**Sizechecker** is a companion tool for **autobrr** that helps prevent accepting new downloads when a certain disk space limit is met. In addition to disk space monitoring, it can send notifications to a Discord webhook if the available disk space falls below a specified threshold.
+**Sizechecker** is a companion tool for **autobrr** that helps prevent accepting new downloads when certain disk space conditions are met. It can check both used and available disk space and can send notifications to a Discord webhook if the specified threshold is crossed.
```

Comment:
I thought about it last time too and didn't have a better alternative but it just dawned on me:
Instead of `prevent accepting` wouldn't it be better to say `rejecting`?

Also, for me a threshold is met and not crossed... maybe opinionated?